rales_engine/app/models/invoice.rb

```diff
@@ -1,3 +1,4 @@
 class Invoice < ApplicationRecord
-  
+  has_many :invoice_items
+  has_many :items, through: :invoice_items
```

Comment:
these ActiveRecord associations look good - be sure to test them though - the `shoulda-matchers` gem is a good way to test your associations!